_import-this-week-in-rails.rb

```diff
@@ -97,6 +97,7 @@ def markdown_render
 title: "#{hey_world_email.title}"
 categories: news
 author: #{hey_world_email.author}
+og_image: assets/images/this-week-in-rails.png
```

Comment:
I presume this is the meta tag we need to include in every markdown document right?

Reply:
Yes, and it will be added by the generators automatically.